.repo-metadata.json

```diff
@@ -1,11 +1,12 @@
 {
     "name": "google-cloud-happybase",
     "name_pretty": "Google Cloud Python HappyBase Client",
-    "client_documentation": "https://google-cloud-python-happybase.readthedocs.io/en/latest/",
-    "release_level": "beta",
+    "client_documentation": "https://cloud.google.com/python/docs/reference/google-cloud-happybase/latest",
+    "release_level": "preview",
     "language": "python",
     "library_type": "INTEGRATION",
     "repo": "googleapis/google-cloud-python-happybase",
     "distribution_name": "google-cloud-happybase",
-    "api_id": "bigtable.googleapis.com"
+    "api_id": "bigtable.googleapis.com",
+    "api_shortname": "google-cloud-happybase"
```

Comment:
This is not a GAPIC library, is the `api_shortname` required?

Reply:
Fixed.